cmd/minikube/cmd/start_flags.go

```diff
@@ -611,6 +611,10 @@ func updateExistingConfigFromFlags(cmd *cobra.Command, existing *config.ClusterC
 		cc.KubernetesConfig.ImageRepository = viper.GetString(imageRepository)
 	}
 
+	if cmd.Flags().Changed("extra-config") {
+		cc.KubernetesConfig.ExtraOptions = config.ExtraOptions;
```

Comment:
Thank you! Please remove the trailing semi-colon so that this passes the lint check.

An integration test would be nice too to prevent future breakage, but I'd be willing to leave it to a follow-up PR.

Reply:
I've fixed the lint error. I've added an integration test and verified that the test fails without the change.